apps/frontend/src/renderer/components/settings/IntegrationSettings.tsx

```
@@ -298,441 +472,120 @@ export function IntegrationSettings({ settings, onSettingsChange, isOpen }: Inte
-                      <Switch
-                        checked={autoSwitchSettings?.proactiveSwapEnabled ?? true}
-                        onCheckedChange={(value) => handleUpdateAutoSwitch({ proactiveSwapEnabled: value })}
-                        disabled={isLoadingAutoSwitch}
-                      />
+                <div className="grid grid-cols-2 gap-3">
+                  <button
+                    onClick={() => handleSelectIntegrationType('oauth')}
+                    disabled={isCreatingOAuth}
+                    className="flex flex-col items-center gap-2 p-4 rounded-lg border-2 border-border hover:border-primary hover:bg-primary/10 transition-colors disabled:opacity-50"
+                  >
+                    <Users className="h-8 w-8 text-primary" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)